Missing deps - react-dom/server - document issue

## 💥 Regression Report

I installed the module, registered it to my fastify server - then starting of my service failed due missing deps.

As written in the readme, I ran `npm i fastify-nextjs next --save` to install this module, package.json diff looked fine:
+    "fastify-nextjs": "^5.1.0",
+    "next": "^10.0.7",

however I tried to start my server which failed:

```
Error: Cannot find module 'react'
Require stack:
- /project/node_modules/next/dist/next-server/server/render.js
- /project/node_modules/next/dist/next-server/server/next-server.js
- /project/node_modules/next/dist/server/next.js
- /project/node_modules/fastify-nextjs/index.js
- /project/src/lib/rest.js
- /project/src/index.js
```

```
Error: Cannot find module 'react-dom/server'
Require stack:
...
```

So I started to install `npm install react --save`, `npm install react-dom/server --save`... but this failed with

```
npm install react-dom/server --save                                  Mi 17 Feb 12:45:35 2021
npm ERR! Can't install github:react-dom/server#9c6de716d028f17736d0892d8a3d8f3ac2cb62bd: Missing package name

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/bar/.npm/_logs/2021-02-17T12_48_12_329Z-debug.log
```

is the manual react and react-dom/server step really needed or just a "following error" of the react-dom/server issue?

## Last working version

As I followed the readme I assume this worked some time ago...

## To Reproduce

Steps to reproduce the behavior:

```js
npm i
npm install react-dom/server --save
```

## Expected behavior

all dependencies are working ;)

## Your Environment

- *node version*: 14.15.4
- *fastify version*: 3.11.0
- *os*: Mac

```
{
  "name": "foo",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "engines": {
    "node": "14.x"
  },
  "scripts": {
    "format": "npm run lint -- --fix",
    "lint": "eslint --ext js,json .",
    "test": "jest",
    "start": "node src/index.js"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "aws-sdk": "^2.839.0",
    "debug": "^4.3.1",
    "dotenv": "^8.2.0",
    "fastify": "^3.11.0",
    "fastify-cookie": "^5.1.0",
    "fastify-jwt": "^2.3.0",
    "fastify-nextjs": "^5.1.0",
    "http-errors": "^1.8.0",
    "next": "^10.0.7",
    "node-schedule": "^2.0.0",
    "pg": "^8.5.1",
    "react": "^17.0.1",
    "simple-event-statistics": "github:neophob/simple-statistics"
  },
  "devDependencies": {
    "eslint": "^7.19.0",
    "eslint-plugin-import": "^2.22.1",
    "eslint-plugin-jest": "^24.1.3",
    "eslint-plugin-json": "^2.1.2",
    "eslint-plugin-react": "^7.22.0",
    "jest": "^26.6.3",
    "node-fetch": "^2.6.1"
  }
}

```

--- a/README.md
+++ b/README.md
@@ -6,7 +6,7 @@ React server side rendering support for Fastify with [Next](https://nextjs.org/d
 
 ## Install
 ```
-npm i fastify-nextjs next --save
+npm i fastify-nextjs next react react-dom --save
 ```
 
 ## Usage

